src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -621,7 +620,14 @@ LRESULT FancyZones::WndProc(HWND window, UINT message, WPARAM wparam, LPARAM lpa
             if (lparam == static_cast<LPARAM>(EditorExitKind::Exit))
             {
                 OnEditorExitEvent();
-                OnDisplayChange(DisplayChangeType::Editor);
+                for (auto& [monitor, zoneWindow] : m_zoneWindowMap)

```

Comment:
Why not to place these changes in `OnDisplayChanged` as it was before? In my opinion it would be clearer if `WndProc` will only have other method calls inside of it.

Reply:
I agree, I can put it in separate method in order to make WndProc as readable as possible, but not in OnDisplayChange. OnDisplayChange does some additional processing (enumerating display monitors) which is not needed when FZ editor is closed.

Reply:
Would it make sense to move it inside `OnEditorExitEvent()`?

Reply:
Yes, makes sense, OnEditorExitEvent parses temporary files filled out by editor and it is a good place to update zone sets for zone windows.